### ISSUE TITLE
Inconsistent Face Tracking Experience on Android Compared to iOS

### ISSUE THREAD

**Issue by bglgwyng** [rodgomesc/vision-camera-face-detector#42]
I have noticed that the face tracking feature on Android is more prone to instability when compared to iOS. It frequently loses track of faces, making it challenging to maintain a consistent bounding box across frames. On the other hand, I find the face tracking on iOS to be reasonably reliable.

I'm curious to know if others have encountered a similar issue. Despite using the same options for the face detector in both versions, I suspect that the inconsistency may stem from the ML Kit library itself.

To investigate further, I plan to assess the quality of face tracking on both Android and iOS using the same image file. I will share the results with you here as soon as possible.

**Comment by salembaira** [comment]
Hi @bglgwyng 

Any progress?

**Comment by bglgwyng** [comment]
@salembaira No. I didn't have time to investigate it. Besides, this issue might be related to the difference between platforms in ML Kit's implementation, rather than this frame processor. Unfortunately, ML Kit is not open-sourced and it makes it very hard to solve this issue.
